styled/styled.go

```diff
@@ -0,0 +1,101 @@
+package styled
+
+import (
+	"fmt"
+)
+
```

Comment:
Add docstring.

Generally, exported symbols should have docstrings, the only exception being docstrings are optional for methods implementing interface.